docs/development_setup.md

```diff
@@ -47,8 +47,9 @@ A minimal database setup for running tests is provided. If the build argument `O
 
 ### Requirements
 
-- Python 3.8+
-- Poetry 1.1.0+
+- Python: 3.8 > 3.11
+- R: ^4.0
+- Poetry: ^1.0
```

Comment:
Why the downgrade in the Poetry version? Wasn't there a reason we required 1.**1**.0 or larger?

Reply:
Good Point. Not sure about the reason but I will revert the change.